test/acceptance/registry.js

```diff
@@ -46,6 +51,136 @@ describe('registry', function(){
     });
   });
 
+  describe('GET /hello-world-custom-headers', function() {
+
+    var httpGetRequest = function(options, callback) {
+      var callbackDone = false;
+      var requestData = url.parse(options.url);
+      requestData.headers = options.headers || {};
+      requestData.method = 'get';
+
+      var req = require('http').request(requestData).on('response', function(response) {
```

Comment:
Completely agree with you. Will modify/fix this once we have the change in minimal-request.